functions/wifi.bash

```diff
@@ -114,3 +114,54 @@ configure_wifi() {
     fi
   fi
 }
+
+
+## Install comitup WiFi hotspot demon
+## Valid arguments: "setup" or "disable"
+## see https://davesteele.github.io/comitup/ppa.html and
+## https://gist.github.com/jjsanderson/ab2407ab5fd07feb2bc5e681b14a537a
+##
+##    setup_hotspot(String option)
+##
+setup_hotspot() {
+
+  if [[ $1 == "install" ]]; then
+    echo -n "$(timestamp) [openHABian] Installing Comitup hotspot... "
+    # get from source - the comitup package in Buster is 2yrs old
+    echo "deb http://davesteele.github.io/comitup/repo comitup main" > /etc/apt/sources.list.d/comitup.list
+    if ! cond_redirect apt-get --quiet update; then echo "FAILED (update apt lists)"; return 1; fi
+
+    if ! cp "${BASEDIR:-/opt/openhabian}"/includes/comitup.conf /etc/comitup.conf; then echo "FAILED (comitup config)"; return 1; fi
+    if cond_redirect apt install --yes -o Dpkg::Options::=--force-confdef comitup; then echo "OK"; else echo "FAILED"; return 1; fi
+    echo "denyinterfaces wlan0 eth0" >> /etc/dhcpcd.conf
```

Comment:
Why deny the interfaces?

Reply:
Good point. 
I tried so much forth and back that I don't remember if I took that off a comitup tutorial or some other hotspot's.
As it works now with that deny in place, comitup must be firing up a DHCP client without use of dhcpcd so it's probably the right thing? not really sure.

Reply:
Same with the dpkg options try to remove it and see if it works. I dislike using code that I don't know what or why is needed.

Reply:
Please test the TestaddHotspot image. Took out both in there, options and denyinterfaces.

Reply:
None of my testing worked with either of these or both take out.
While there's a smallish chance that there was a package change in the last days that causes these problems, I don't think so (no indication of that on the failing box).
So let's leave both of these in.
Will create another addHotspot (not TestaddHotspot) image to confirm